.idea/compiler.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@sladomic Don't include IDEA files in the versioning system. Add `./idea` to the `.gitignore` file to prevent this.